src/ets/scala/io/iohk/ethereum/ets/blockchain/ScenarioSetup.scala

```diff
@@ -22,16 +22,16 @@ import scala.util.{Failure, Success, Try}
 
 object ScenarioSetup {
   val testContext = Scheduler.fixedPool("scenario-setup-pool", 4)
-  val specificConfig = ethash.EthashConfig(Config.config)
+  val specificConfig = pow.EthashConfig(Config.config)
   val fullConfig = FullConsensusConfig(ConsensusConfig(Config.config), specificConfig)
 
   def loadEthashConsensus(
       vm: VMImpl,
       blockchain: BlockchainImpl,
       blockchainConfig: BlockchainConfig,
       validators: ValidatorsExecutor
-  ): ethash.EthashConsensus = {
-    val consensus = EthashConsensus(vm, blockchain, blockchainConfig, fullConfig, validators, NoAdditionalEthashData)
+  ): pow.PoWConsensus = {
```

Comment:
`pow` and `pow.PoWConsensus` are both imported, I suggest to simply write `PoWConsensus`

Reply:
Fixed